src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +130,661 @@ export class JailAddComponent implements OnInit {
+                  parent jails have access to the jail objects but \
+                  <i>not</i> private keys. <br>\
+                  <b>Disable</b>: The jail cannot perform any \
+                  <b>sysvshm</b> related system calls.'),
+        options: [{
+            label: 'Inherit',
+            value: 'inherit',
+        }, {
+            label: 'New',
+            value: 'new',
+        }, {
+            label: 'Disable',
+            value: 'disable',
+        }]
+    },
+    {
       type: 'checkbox',
       name: 'allow_set_hostname',
-      placeholder: T('allow_set_hostname'),
-      tooltip: T('Allow the jail hostname to be changed with <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=hostname&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">hostname(1)</a> or <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=sethostname&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">sethostname(3)</a>.'),
-    }, {
+      placeholder: T('allow.set_hostname'),
+      tooltip: T('Allow the jail hostname to be changed with <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=hostname&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">hostname(1)</a> or <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=sethostname&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">sethostname(3)</a>.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_sysvipc',
-      placeholder: T('allow_sysvipc'),
-      tooltip: T('In FreeBSD 11.0 and later, this setting is deprecated.\
- Use <b>sysvmsg</b>, <b>sysvsem</b>, and <b>sysvshm</b> instead. Choose\
- if a process in the jail has access to System V IPC primitives.'),
-    }, {
+      placeholder: T('*allow.sysvipc'),
+      tooltip: T('Choose if a process in the jail has access to \
+                  System V IPC primitives. Equivalent to setting \
+                  sysvmsg, sysvsem, and sysvshm to <b>Inherit</b>. \
+                  <b>*Deprecated in FreeBSD 11.0 and later!</b><br> \
+                  Use <b>sysvmsg</b>, <b>sysvsem</b>, and \
+                  <b>sysvshm</b> instead.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_raw_sockets',
-      placeholder: T('allow_raw_sockets'),
-      tooltip: T('Select this to allow utilities like <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=ping&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">ping(8)</a> and <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=traceroute&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">traceroute(8)</a> to operate inside the jail. When\
- checked, the source IP addresses are enforced to comply with the IP\
- address bound to the jail, ignoring the the IP_HDRINCL flag on the\
- socket.'),
-    }, {
+      placeholder: T('allow.raw_sockets'),
+      tooltip: T('Set to allow utilities like <a \
```

Comment:
"Set to allow raw sockets. Utilities like <a>ping</a> and <a>traceroute</a> require raw sockets. When set, ..."

Reply:
fixed